modules/react/button/lib/Hyperlink.tsx

```diff
@@ -15,66 +16,59 @@ export interface HyperlinkProps {
   children?: React.ReactNode;
 }
 
-const variantStyles = {
-  inverse: {
-    color: colors.frenchVanilla100,
-    '&:hover': {
-      color: colors.frenchVanilla100,
-      background: 'rgba(255, 255, 255, 0.1)',
-    },
-    '&:focus': {
-      boxShadow: `0 0 0 2px ${colors.frenchVanilla100}`,
-    },
-    '&:active': {
-      color: colors.blueberry600,
-      background: colors.soap200,
-    },
-  },
-};
-
-const anchorVariants = (props: HyperlinkProps) => {
-  if (props.variant === 'inverse') {
-    return variantStyles.inverse;
-  }
-  return {};
-};
-
-const Anchor = styled('a')<HyperlinkProps & StyledType>(
-  {
-    fontFamily: type.properties.fontFamilies.default,
+const hyperlinkStencil = createStencil({
+  base: {
+    fontFamily: system.fontFamily.default,
     textDecoration: 'underline',
-    color: colors.blueberry400,
+    color: system.color.text.primary.default,
     cursor: 'pointer',
-    borderRadius: borderRadius.s,
+    borderRadius: system.shape.x1,
     display: 'inline-block',
     padding: '0 2px',
     margin: '0 -2px',
     transition: 'color 0.15s,background-color 0.15s',
     '&:hover': {
-      color: colors.blueberry500,
-      background: colors.soap200,
+      color: system.color.text.primary.strong,
+      background: system.color.bg.alt.soft,
     },
     '&:focus': {
```

Comment:
it won't let me comment on a line you can't changed, but we need to add `&.focus, &:focus-visible` on line 34.  This ensures that we apply focus styles weather someone adds the className of `focus` or using the browsers heuristics of determining when to add focus which is [focus visible](https://developer.mozilla.org/en-US/docs/Web/CSS/:focus-visible).

Reply:
```tsx
'&:focus, &.focus, &:focus-visible': {
      boxShadow: `0 0 0 2px ${cssVar(system.color.bg.primary.default)}`,
      outline: 'none',
    },
```